projects/packages/videopress/src/client/block-editor/blocks/video/components/tracks-control/track-form.tsx

```diff
@@ -76,6 +78,18 @@ export default function TrackForm( {
 		onSave( track );
 	}, [ track ] );
 
+	const setSourceLanguage = useCallback( ( newSrcLang: string ) => {
+		updateTrack( 'srcLang', newSrcLang );
+
+		if ( newSrcLang?.length > 5 ) {
+			return setError(
+				__( 'Language must be five characters or less.', 'jetpack-videopress-pkg' )
```

Comment:
Maybe it's good to reference the field by the full label given that the notice is not near the field.

```suggestion
				__( 'Source Language must be five characters or less.', 'jetpack-videopress-pkg' )
```

Reply:
yeah, I removed the `Source ` to avoid two lines :-/